### PR TITLE
Added Sematext as partner

### PR DESCRIPTION
**Pull request by otisg**
### Description
Added Sematext as partner
 
### Issues Resolved
[List any issues this PR will resolve]

### Check List
- [x] Commits are signed per the DCO using --signoff

By submitting this pull request, I confirm that my contribution is made under the terms of the BSD-3-Clause License.

### CODE REVIEW

File: _partners/sematext.md [hunk 1]
```diff
@@ -0,0 +1,81 @@
+---
+name: Sematext
+# name_long is used on the partner details page.
+name_long: 'Sematext Group'
+# upload your logo to the following directory - must be square
+logo: '/assets/media/partners/sematext.png'
```
Comment (nateynateynate) [inline _partners/sematext.md:6]: I think you meant `octi-square-logo.png`

File: _partners/sematext.md [hunk 2]
```diff
@@ -0,0 +1,81 @@
...
+link: 'https://sematext.com'
+# logo_large is used on the partner details page.
+logo_large: '/assets/media/partners/sematext.png'
+product_image: '/assets/media/partners/sematext.png'
```
Comment (nateynateynate) [inline _partners/sematext.md:10]: octi-square-logo.png

File: _partners/sematext.md [hunk 3]
```diff
@@ -0,0 +1,81 @@
...
+description: 'Sematext has deep expertise with OpenSearch and building search applications on top of it.  We've helped 100s of organizations with OpenSearch, Elasticsearch, and Solr.
```
Comment (nateynateynate) [inline _partners/sematext.md:11]: You probably want to double-quote this so that the all of the `We've`s don't prematurely end your description.

**Review by nateynateynate — CHANGES_REQUESTED** [review]
Let's get the quoting issues and logo issue addressed and then we can get this merged with a quickness.

File: _partners/sematext.md (hunk 1 above)
Comment (otisg) [reply _partners/sematext.md:6]: Ah, right, thanks.  Re-added with the correct name now.

File: _partners/sematext.md (hunk 3 above)
Comment (otisg) [reply _partners/sematext.md]: Thanks.  Reworded to avoid the problem.

**Review by nateynateynate — APPROVED** [review]
Perfect! Thanks so much @otisg! Thanks for partnering with us!

**Comment by otisg** [comment]
Thanks @nateynateynate for being so quick!  I assume we should show up on https://opensearch.org/partners/ in N hours or days?

**Comment by nateynateynate** [comment]
> Thanks @nateynateynate for being so quick! I assume we should show up on https://opensearch.org/partners/ in N hours or days?

CI/CD should take around 20 minutes. Enjoy! :-)

**Pull request merged**